main.tf

```diff
@@ -1,73 +1,40 @@
 # Using tf_admin module
 
-# Apply the tf_label module for this resource
-module "label" {
-  source    = "git::https://github.com/cloudposse/tf_label.git?ref=tags/0.1.0"
-  namespace = "${var.namespace}"
-  stage     = "${var.stage}"
-  name      = "${var.name}"
+resource "aws_security_group_rule" "ssh" {
+  from_port         = 22
+  protocol          = "-1"
+  security_group_id = "${module.instance.security_group.id}"
```

Comment:
There is no `security_group` output in `terraform-aws-ec2-instance`.
You should use `security_group_ids` output.